submarine-server/server-core/src/main/java/org/apache/submarine/server/rest/EnvironmentRestApi.java

```diff
@@ -159,6 +159,30 @@ public Response listEnvironment(@QueryParam("status") String status) {
     }
   }
 
+  /**
+   * List all environments from database.
+   * @return environment list
+   */
+  @GET
+  @Path("/list")
+  @Operation(summary = "List of Environments from database",
+          tags = {"environments"},
+          responses = {
+                  @ApiResponse(description = "successful operation", 
+                      content = @Content(
+                          schema = @Schema(
+                              implementation = Environment.class)))})
+  public Response listAllEnvironments() {
```

Comment:
There is a `listEnvironment` in `EnvironmentRestApi.java`, why create a new function `listAllEnvironments`?
https://github.com/apache/submarine/blob/cc9dde59564a58dcbcdc22a183e88fb2a90cdd6f/submarine-server/server-core/src/main/java/org/apache/submarine/server/rest/EnvironmentRestApi.java#L151-L159

Reply:
`listEnvironment` just get those environments in cache. `listAllEnvironments` will get environments from database.

Reply:
I think we can merge it rather than create new.

Reply:
I will merge it.

Reply:
@kobe860219  Thanks for working on this. Even from API end point perspective, I don't see a need for "list". Just typing "/v1/environment" should return the list of environments. Change is required only in model layer.

Reply:
@jiwq @pingsutw @manirajv06  I have updated this pr. Please take a look, Thanks :))